index.html

```diff
@@ -292,7 +292,7 @@ <h3>
               </td>
             </tr>
             <tr>
-              <td><a href="https://json-web-proofs.github.io/json-web-proofs/">JSON Web Proof (JWP)</a></td>
+              <td><a href="https://json-web-proofs.github.io/json-web-proofs/">VC-JSON Web Proof (JWP)</a></td>
```

Comment:
@brentzundel @Sakurann -- just making sure both of you see this. We are standardizing VC-JWP in this group, NOT JWP. JWP will be standardized in IETF. We probably need to make this much more clear in the charter.